common/jsonify.py

```diff
@@ -14,6 +13,9 @@ def default(self, obj):
             return obj.strftime('%H:%M')
         elif isinstance(obj, set):
             return list(obj)
+        elif callable(obj):
```

Comment:
smart. do we serialise user object to json anywhere?

Reply:
This is actually for `User.is_authenticated` that used to be a function and from 1.10 is a property that returns a `BoolCallable` - which works as a property and a method, so it's a transparent change when used in Python, but breaks serialization. More info here: https://docs.djangoproject.com/en/2.1/releases/1.10/#using-user-is-authenticated-and-user-is-anonymous-as-methods

Reply:
cool, but where is it used? sessions maybe?

Reply:
The template causing the issue was `templates/p3/base.html` in the following lines:
```
var user = {
            'authenticated': {{ user.is_authenticated|json_ }}
        }
```
The user object seems to be used in `schedule.js` file.

Reply:
Let's keep it that way then, something to address in the future.